dist/src/main/java/io/camunda/zeebe/shared/management/ExportingEndpoint.java

```diff
@@ -21,6 +21,7 @@
 @WebEndpoint(id = "exporting")
 public final class ExportingEndpoint {
   static final String PAUSE = "pause";
+  static final String SOFT_PAUSE = "softPause";
```

Comment:
:x: I was thinking the endpoint should be something like `exporting/pause?soft=true`. What do you think?

Reply:
Having this as a separate endpoint helps with the testing in ExportingEndpointTest, and also makes the endpoints in line with the ones exposed on the broker. Do you think it is worth it to change here?

Reply:
You can still test it easily in ExportingEndpointTest. I don't see any problem.

There is no need to match the user facing API to match the internal implementation.